README.md

```diff
@@ -1,5 +1,11 @@
 # pyramid-learning-journal
 
+<h1>Step 2</h1>
+
+Step 2 w/ Carlos partnered on the journal server project. Implemented journal entries and incorporated templating to streamline user experience and refactor unnecessary code.
+
+During install, run "$pip install -e ." to install needed dependences.
+
 <h1>Step 1</h1>
 
-Step 1 partner programming. Using Pyramid and Heroku.
+Step 1 partner programming. Using Pyramid and Heroku. One learning journal is attached as part of the lab.
```

Comment:
Good to see you maintaining a readme. Keep this up to date and useful for someone new to your codebase.